src/OSPSuite.Core/Domain/Services/EventBuilderTask.cs

```diff
@@ -157,5 +169,24 @@ private IEnumerable<IContainer> sourceContainersFor(TransportBuilder transport,
       {
          return allEventGroupParentChildContainers.AllSatisfiedBy(transport.SourceCriteria);
       }
+
+      private void tryMergeEventGroupInContainer(IContainer mergeContainer, EventGroup eventGroup, MergeBehavior mergeBehavior, EventGroupBuildingBlock eventGroupBuildingBlock)
+      {
+         //probably should never happen
+         if (mergeContainer == null)
+            return;
+
+         try
+         {
+            if (mergeBehavior == MergeBehavior.Extend)
+               _containerMergeTask.AddOrMergeContainer(mergeContainer, eventGroup);
+            else
+               _containerMergeTask.AddOrReplaceInContainer(mergeContainer, eventGroup);
+         }
+         catch (Exception)
+         {
+            throw new OSPSuiteException(Error.CannotFindParentContainerWithPath(mergeContainer.ParentPath.PathAsString, mergeContainer.Name, eventGroupBuildingBlock.Name, eventGroupBuildingBlock.Module.Name));
+         }
```

Comment:
I don't understand why we need the try catch here and also the exception message seems wrong?
(`mergeContainer.ParentPath` is not involved here at all)

Reply:
Removed. That was only needed in case there was some unresolved `ParentPath` as you pointed out. This cannot (yet) happen for events :)